.travis.yml

```diff
@@ -4,7 +4,7 @@ os:
   - linux
   - osx
 julia:
-  - 0.6
+  - 1.0
```

Comment:
Can you add 0.7 back (in both the CI scripts and in REQUIRE)? It's easier to migrate packages using 0.7 than 1.0 (you get all these lovely depwarns...), and since the two are compatible there is no reason to exclude 0.7.